### PR TITLE
Add email confirming places at SR2020

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Fixes https://github.com/srobo/tasks/issues/307

Replaces #27

### CODE REVIEW

File: SR2020/2019-09-25-confirm-competition-place.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+subject: Confirming your place at SR2020!
+---
+
+Hello there!
+
+We're pleased to confirm your place at Student Robotics 2020! You will receive an invite shortly to the mailing list on Google Groups for this competition cycle.
+
+With this, we're also happy to announce the dates and locations for kickstart. Kickstart is the event which kicks off a competition year, where the game will be announced and robotics kits will be handed out. The event is also invaluable to competitors, allowing them to discuss ideas, and get direct input and support from our talented blue shirts.
+
+Kickstart will be on 26th October 2019
```
[the remark this answers is not in the document]
Comment (RealOrangeOne) [reply SR2020/2019-09-25-confirm-competition-place.md]: That's all on the event specific pages, which I think is fine?

File: SR2020/2019-09-25-confirm-competition-place.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This year, we're running 3 Kickstart events:
```
[the remark this answers is not in the document]
Comment (RealOrangeOne) [reply SR2020/2019-09-25-confirm-competition-place.md]: I fail to see the difference?

File: SR2020/2019-09-25-confirm-competition-place.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Adimote) [reply SR2020/2019-09-25-confirm-competition-place.md]: Sure :+1:

File: SR2020/2019-09-25-confirm-competition-place.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Adimote) [reply SR2020/2019-09-25-confirm-competition-place.md]: one of them reads slightly nicer, I'm happy for this not to go in (hence the [nit])

**Comment by RealOrangeOne** [comment]
This has now been sent.

**Pull request merged**